Mathlib/Topology/Compactification/OnePointEquiv.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2024 Bjørn Kjos-Hanssen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bjørn Kjos-Hanssen
+-/
+import Mathlib.Data.Real.Basic
+import Mathlib.LinearAlgebra.Projectivization.Basic
+import Mathlib.Topology.Compactification.OnePoint
+
+/-!
+# OnePoint Compactification and Projectivization
+
+We construct a set-theoretic equivalence between
+`OnePoint K` and the projectivization `ℙ K (Fin 2 → K)`
+for an arbitrary field `K`.
+
+(This equivalence can be extended to a homeomorphism
+in the case `K = ℝ`, where `OnePoint ℝ` gets the topology of one-point compactification.
+This result is to be added in a different file.)
+
+## Main definitions
+
+* `divOnePoint`: Division on `K` taking values in `OnePoint K`.
+* `divSlope`: Lifting `divOnePoint` to a map `ℙ K (Fin 2 → K) → OnePoint K`.
+
+## Main results
+
+* `ℙ K (Fin 2 → K) ≃ OnePoint K`
+
+## Tags
+
+one-point extension, projectivization
+-/
+
+
+open scoped LinearAlgebra.Projectivization OnePoint
+open Projectivization Classical
+
+/-!
+### Definition and basic properties
+
+In this section we define `divOnePoint` and prove that it lifts to `divSlope`.
+-/
+
+/-- A modified division from a `DivisionRing` to its `OnePoint` extension,
+so as to satisfy `1 ÷ 0 = ∞`. -/
+noncomputable def divOnePoint {K : Type} [DivisionRing K] (a : K) (r : K): OnePoint K :=
+  ite (r ≠ 0) (a / r) ∞
+
+/-- Uncurried version of `divOnePoint`, with nonzeroness assumption. -/
+noncomputable def divOnePoint' {K : Type} [DivisionRing K]
+  (u : {v : Fin 2 → K // v ≠ 0}) : OnePoint K :=
+  if u.1 1 ≠ 0 then some (u.1 0 / u.1 1) else ∞
+
+/-- Notation `÷` showing that `div_onePoint` is distinct from
+the ordinary division `/` of a `DivisionRing`. -/
+infix:50 " ÷ " => divOnePoint
```

Comment:
Can you please mention this notation above, both in the module docstring and the definition of `divOnePoint`?